fix(community-edition-setup): oxAuthzChallSess misses exp and del columns

## Describe the issue

```
2025-04-16 14:53:03,528 ERROR [ForkJoinPool.commonPool-worker-1] [org.gluu.oxauth.service.CleanerTimer] (CleanerTimer.java:217) - Failed to perform clean up.
org.gluu.persist.exception.EntryDeleteException: Failed to convert filter '(&(del=true)(exp<=2025-04-16T14:53:03.521))' to expression
	at org.gluu.persist.sql.impl.SqlEntryManager.removeImpl(SqlEntryManager.java:355) ~[gluu-orm-sql-4.5.9-SNAPSHOT.jar:?]
	at org.gluu.persist.sql.impl.SqlEntryManager.remove(SqlEntryManager.java:322) ~[gluu-orm-sql-4.5.9-SNAPSHOT.jar:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:580) ~[?:?]
	at org.jboss.weld.bean.proxy.AbstractBeanInstance.invoke(AbstractBeanInstance.java:38) ~[weld-core-impl-3.1.9.Final.jar:3.1.9.Final]
	at org.jboss.weld.bean.proxy.ProxyMethodHandler.invoke(ProxyMethodHandler.java:106) ~[weld-core-impl-3.1.9.Final.jar:3.1.9.Final]
	at org.gluu.persist.PersistenceEntryManager$EntityManager$1948487569$Proxy$_$$_WeldClientProxy.remove(Unknown Source) ~[gluu-orm-core-4.5.9-SNAPSHOT.jar:?]
	at org.gluu.oxauth.service.CleanerTimer.cleanup(CleanerTimer.java:213) ~[classes/:?]
	at org.gluu.oxauth.service.CleanerTimer$Proxy$_$$_WeldSubclass.cleanup(Unknown Source) ~[classes/:?]
	at org.gluu.oxauth.service.CleanerTimer.processImpl(CleanerTimer.java:170) ~[classes/:?]
	at org.gluu.oxauth.service.CleanerTimer$Proxy$_$$_WeldSubclass.processImpl(Unknown Source) ~[classes/:?]
	at org.gluu.oxauth.service.CleanerTimer.process(CleanerTimer.java:118) ~[classes/:?]
	at org.gluu.oxauth.service.CleanerTimer$Proxy$_$$_WeldSubclass.process$$super(Unknown Source) ~[classes/:?]
	at jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:580) ~[?:?]
	at org.jboss.weld.interceptor.proxy.TerminalAroundInvokeInvocationContext.proceedInternal(TerminalAroundInvokeInvocationContext.java:51) ~[weld-core-impl-3.1.9.Final.jar:3.1.9.Final]
	at org.jboss.weld.interceptor.proxy.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:78) ~[weld-core-impl-3.1.9.Final.jar:3.1.9.Final]
	at org.gluu.service.cdi.async.AsynchronousInterceptor$1.get(AsynchronousInterceptor.java:36) ~[oxcore-service-4.5.9-SNAPSHOT.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:387) ~[?:?]
	at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1312) ~[?:?]
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1843) ~[?:?]
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1808) ~[?:?]
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:188) ~[?:?]
Caused by: org.gluu.persist.exception.operation.SearchException: Unknown column name 'del' in table/child table 'oxAuthzChallSess'
	at org.gluu.persist.sql.impl.SqlFilterConverter.getAttributeType(SqlFilterConverter.java:420) ~[gluu-orm-sql-4.5.9-SNAPSHOT.jar:?]
	at org.gluu.persist.sql.impl.SqlFilterConverter.isMultiValue(SqlFilterConverter.java:389) ~[gluu-orm-sql-4.5.9-SNAPSHOT.jar:?]
	at org.gluu.persist.sql.impl.SqlFilterConverter.convertToSqlFilterImpl(SqlFilterConverter.java:182) ~[gluu-orm-sql-4.5.9-SNAPSHOT.jar:?]
	at org.gluu.persist.sql.impl.SqlFilterConverter.convertToSqlFilterImpl(SqlFilterConverter.java:119) ~[gluu-orm-sql-4.5.9-SNAPSHOT.jar:?]
	at org.gluu.persist.sql.impl.SqlFilterConverter.convertToSqlFilter(SqlFilterConverter.java:85) ~[gluu-orm-sql-4.5.9-SNAPSHOT.jar:?]
	at org.gluu.persist.sql.impl.SqlFilterConverter.convertToSqlFilter(SqlFilterConverter.java:80) ~[gluu-orm-sql-4.5.9-SNAPSHOT.jar:?]
	at org.gluu.persist.sql.impl.SqlEntryManager.toSqlFilterWithEmptyAlias(SqlEntryManager.java:811) ~[gluu-orm-sql-4.5.9-SNAPSHOT.jar:?]
	at org.gluu.
```


https://chat.gluu.org/group/product_gluu4?msg=JrHGDsPvSWvGEbxeo

## Expected behavior
`del` and `exp` columns must be present in oxAuthzChallSess  OC

--- a/community-edition-setup/schema/gluu_schema.json
+++ b/community-edition-setup/schema/gluu_schema.json
@@ -5453,7 +5453,9 @@
                 "oxAuthUserDN",
                 "creationDate",
                 "oxAuthClientId",
-                "oxAttributes"
+                "oxAttributes",
+                "exp",
+                "del"
             ],
             "must": [
                 "objectclass"

